scripts/install.sh

```diff
@@ -209,6 +210,7 @@ fi
 
 # Trigger installation.
 action "Installing package into ${prefix}"
+insert_path="$sudo echo 'export PATH=\$PATH:/opt/tenzir/bin' > /etc/profile.d/tenzir.sh"
```

Comment:
I'm not sure how to achieve this on macOS.

Reply:
There's no equivalent. Shells have their specific way of doing this, e.g., via `.bash_profile`.

Reply:
Can we create a symlink that points from `/usr/local/bin/tenzir` to `/opt/tenzir/bin/tenzir`?